### DISCUSSION TITLE
Desktop shortcuts

### DISCUSSION THREAD

**Discussion by wehkah** [osmandapp/OsmAnd#11448] [Ideas]
Hello,

would it be possible to place a shortcut for a location on the android desktop, that will open Osmand and start routing to that location without the need for more taps? The shortcut should also include the routing profile that is to be used (car, bike, etc).

**Comment by vshcherb** [comment]
Yes, we plan to make something similar https://github.com/osmandapp/OsmAnd/issues/6555
